### ISSUE TITLE
 error on state_location_payload_key in version 0.5.2

### ISSUE THREAD

**Issue by FlyingDiver** [FlyingDiver/Indigo-Shims#21]
https://forums.indigodomo.com/viewtopic.php?f=316&t=25195

**Comment by FlyingDiver** [comment]
```
MQTT Shims Error Ring Alarm Basestation Info: error on state_location_payload_key, aborting
MQTT Shims Error Ring Alarm Keypad Entryway: error on state_location_payload_key, aborting
MQTT Shims Error Ring Alarm Keypad Bedroom: error on state_location_payload_key, aborting
MQTT Shims Error Ring Alarm Keypad Basement: error on state_location_payload_key, aborting
MQTT Shims Error Ring Alarm ZWave Extender: error on state_location_payload_key, aborting
MQTT Shims Error Ring Alarm ZWave Extender Basement: error on state_location_payload_key, aborting
```

**Comment by FlyingDiver** [comment]
```
   MQTT Shims                      
message_type: '##RingAlarm##'
props:
    SupportsBatteryLevel: true
    battery_payload_key: topics.info.topics.state.payload.batteryLevel
    state_dict_payload_key: topics.info.topics.state.payload
    state_location: payload
    state_location_payload_type: json
    uid_location: topic
    uid_location_topic_field: '3'
trigger:
    match_list: '["Match: ring", "Match:ACCOUNT_GUID, "Match: alarm", "Any: ", "End: "]'
    queueMessage: true
type: shimGeneric
```

**Comment by FlyingDiver** [comment]
```
   MQTT Connector Debug            Ring MQTT Broker: Message topic: ring/ACCOUNT_GUID/alarm/DEVICE_GUID/info/state, payload = {"acStatus":"error","batteryLevel":85,"batteryStatus":"ok","brightness":1,"chirps":"enabled","commStatus":"ok","firmwareStatus":"up-to-date","lastCommTime":"2021-04-30T19:01:09Z","lastUpdate":"2021-04-30T19:01:09Z","linkQuality":"ok","powerSave":"off","serialNumber":"SER_NUM","tamperStatus":"ok","volume":0.7}
   MQTT Connector Debug            Ring MQTT Broker: processReceivedMessage: ring/ACCOUNT_GUID/alarm/DEVICE_GUID/info/state, payload: {"acStatus":"error","batteryLevel":85,"batteryStatus":"ok","brightness":1,"chirps":"enabled","commStatus":"ok","firmwareStatus":"up-to-date","lastCommTime":"2021-04-30T19:01:09Z","lastUpdate":"2021-04-30T19:01:09Z","linkQuality":"ok","powerSave":"off","serialNumber":"SER_NUM","tamperStatus":"ok","volume":0.7}
   MQTT Connector Debug            Ring MQTT Broker: agg_dict: {u'topics': {u'info': {'topics': {u'state': {u'payload': {u'batteryStatus': u'ok', u'lastUpdate': u'2021-04-30T19:01:09Z', u'serialNumber': u'SER_NUM', u'brightness': 1, u'chirps': u'enabled', u'firmwareStatus': u'up-to-date', u'acStatus': u'error', u'lastCommTime': u'2021-04-30T19:01:09Z', u'volume': 0.7, u'tamperStatus': u'ok', u'linkQuality': u'ok', u'batteryLevel': 85, u'commStatus': u'ok', u'powerSave': u'off'}}}}}}
   MQTT Connector Debug            Ring MQTT Broker: Saved states, topic: ring/ACCOUNT_GUID/alarm/DEVICE_GUID, payload: {"topics": {"info": {"topics": {"state": {"payload": {"batteryStatus": "ok", "lastUpdate": "2021-04-30T19:01:09Z", "brightness": 1, "chirps": "enabled", "serialNumber": "SER_NUM", "firmwareStatus": "up-to-date", "acStatus": "error", "lastCommTime": "2021-04-30T19:01:09Z", "volume": 0.7, "tamperStatus": "ok", "linkQuality": "ok", "batteryLevel": 85, "commStatus": "ok", "powerSave": "off"}}}}, "status": {"payload": "online"}}}
```

**Comment by FlyingDiver** [comment]
```
   MQTT Connector                  1416305691:ring/ACCOUNT_GUID/alarm/DEVICE_GUID   ring/ACCOUNT_GUID/alarm/DEVICE_GUID
{
    "topics": {
        "info": {
            "topics": {
                "state": {
                    "payload": {
                        "acStatus": "error",
                        "batteryLevel": 85,
                        "batteryStatus": "ok",
                        "brightness": 1,
                        "chirps": "enabled",
                        "commStatus": "ok",
                        "firmwareStatus": "up-to-date",
                        "lastCommTime": "2021-04-30T19:01:09Z",
                        "lastUpdate": "2021-04-30T19:01:09Z",
                        "linkQuality": "ok",
                        "powerSave": "off",
                        "serialNumber": "SER_NUM",
                        "tamperStatus": "ok",
                        "volume": 0.7
                    }
                }
            }
        },
        "status": {
            "payload": "online"
        }
    }
}
```

**Comment by FlyingDiver** [comment]
Fixed in 0.5.6.
